dev-docs/source/GitWorkflow.rst

```diff
@@ -85,6 +85,7 @@ When creating a pull request you should:
 
   - The title should **not** contain the issue number
 - `Reference the issue which the pull request is closing  <https://github.com/blog/1506-closing-issues-via-pull-requests>`_, using one of `these <https://help.github.com/articles/closing-issues-via-commit-messages>`_ keywords
+- State the user who initiated the original issue, if they are named in the issue. Please do not put full email addresses on the Pull Request, as it is publicly accessible.
```

Comment:
Is it worth adding the facility too? It might not be obvious to the person picking up the ticket how to get in touch.

Reply:
Good point - done.